sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -1007,18 +1007,129 @@ test_suite(
     ],
 )
 
-opentitan_functest(
-    name = "shutdown_watchdog",
-    srcs = ["hang_test.c"],
-    cw310 = cw310_params(
-        exit_success = "Returning after 5 seconds",
-    ),
-    local_defines = [
-        "HANG_SECS=5",
-    ],
-    targets = ["cw310_rom"],
-    deps = [
-        "//sw/device/lib/testing/test_framework:ottf_main",
+SHUTDOWN_WATCHDOG_BITE_THRESHOLDS = [
+    400000,  # 2 seconds at 200 kHz
```

Comment:
nit: Is there a way we can encode this in ms or us and convert to cycles based on the target (sim, fpga, silicon)?

Reply:
Yes, I think so. We can add platform specific constants to `CONST` and define macros to handle the conversions for us. The issue, however, is that different platforms are handled in `opentitan_functest()` and defining platform-specific OTP images here would leak target naming details here as well. Since we currently run this only on CW310, I hope we can tackle this when we enable this test for other platforms. Created #16016.